### PR TITLE
[MBL-1667] String Translations

### PR DESCRIPTION
**Pull request by scottkicks**
# 📲 What

Pulls in the latest strings translations and replaces our hard coded strings accordingly.
- Also sneaks in a quick change so that the `configureEstimatedShippingView` returns optional Strings and some extra cleanup
- If you feel that these changes should be kept separate in their own PR lmk.

# 🤔 Why

KS supports multiple countries.

# 🛠 How

run `make strings`
update hardcoded strings

### CODE REVIEW

**Comment by scottkicks** [comment]
@ifosli ive reached out about whether displaying values in a min-max format is appropriate for all languages. We believe that it's fine or else the translations team likely would have called that out when translating. 

I'll handle any updates needed if they come back saying we need them.

**Comment by nativeksr** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by nativeksr** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: Kickstarter-iOS/Features/ConfirmDetails/Controllers/NoShippingConfirmDetailsViewController.swift [hunk 1]
```diff
@@ -80,7 +80,7 @@ final class NoShippingConfirmDetailsViewController: UIViewController, MessageBan
 
   private var sessionStartedObserver: Any?
 
-  private let viewModel: ConfirmDetailsViewModelType = ConfirmDetailsViewModel()
```
Comment (scottkicks) [inline Kickstarter-iOS/Features/ConfirmDetails/Controllers/NoShippingConfirmDetailsViewController.swift:83]: Turns out this view controller hasn't been using the right view model 😅

**Pull request closed without merging**